rocAL/rocAL_pybind/amd/rocal/fn.py

```diff
@@ -0,0 +1,562 @@
+
```

Comment:
please remove all unused imports and fix Codacy errors

Reply:
The fn.py requires the readers , decoders to be imported from readers.py / decoders.py so that it can be accessed as
fn.readers.function() in example scripts

But we don't require to use the readers / decoders inside the fn.py file 
Its for accessing similar to DALI